README.md

```diff
@@ -299,3 +299,15 @@ spec:
           userDataSecret:
             name: windows-user-data
 ```
+### Windows nodes k8s component upgrade
+
+When a new version of WMCO is released that is compatible with the current cluster version, an update can take place.
```

Comment:
`, an operator upgrade will take place which will result in the Kubernetes components in the Windows machine to be upgraded.`